config.js

```diff
@@ -26,14 +26,14 @@ module.exports = {
   },
   login: {
     tokenExpiry: 1800,
-    appPath: '/acrs/select-form',
+    appPath: '/acrs/start',
     invalidTokenPath: '/acrs/token-invalid',
     allowSkip: String(process.env.ALLOW_SKIP) === 'true',
     skipEmail: process.env.SKIP_EMAIL
   },
   sessionDefaults: {
-    steps: ['/select-form', '/confirm', '/confirmation'],
-    fields: ['brp', 'uan', 'date-of-birth', 'csrf-secret', 'errorValues', 'errors']
+    steps: ['/start', '/select-form', '/information-you-have-given-us', '/who-is-completing-form'],
+    fields: ['user-email', 'id-type', 'date-of-birth', 'csrf-secret', 'errorValues', 'errors']
```

Comment:
I think you'll want to keep brp, uan, date-of-birth and id-type in here so they aren't cleaned out.

Reply:
This `fields` array is what session items are not cleared down from the verify session. My concern is that we lose data in some login scenarios when the data is not carried over.